app/models/user.rb

```diff
@@ -1,3 +1,10 @@
 class User < ActiveRecord::Base
   include Clearance::User
+
+  has_many :reviews
+
+  def can_review_book?(book)
+    # need to eager load review or put this logic somewhere else
+    !reviews.pluck(:book_id).include? book.id
```

Comment:
This queries for the `book.reviews` again, it's already loaded by the controller. I was considering putting this in a `BookDecorator` of some kind, but I'm not sure where the best place for the logic is. Should a book know if a user can add a review?